core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -191,4 +191,384 @@ public UnknownException(Throwable cause) {
             );
         }
     }
+
+    /**
+     * Exception thrown by AWS Cognito Auth.
+     */
+    public static class AWSCognitoAuthException extends AuthException {
```

Comment:
We shouldn't put this in the `core` module, since it has "Cognito" in the name. Instead, this should live in the `aws-auth-cognito` module, somewhere.

Reply:
Got it, a real big lesson😂

Reply:
Haha :-D